src/endpoints/user/getUserHobby.ts

```diff
@@ -0,0 +1,19 @@
+import {IncomingMessage, ServerResponse} from 'http'
+import { UserService } from '../../userService';
+import { getUserIdFromRequest } from '../../utils';
+import { PATHS } from '../../constants';
+
+export function getUserHobbyRoute(req: IncomingMessage, res: ServerResponse) {
+    const userService = UserService.getInstance();
+    const userId: number = getUserIdFromRequest(req, res, PATHS.userHobby);
+    const userHobby = userService.getUserHobbies(userId);
+
+    if (userHobby && userHobby.length) {
+        res.statusCode = 200;
+        res.end(JSON.stringify(userHobby));
+        return;
+    } else {
+        res.statusCode = 204;
```

Comment:
If hobby not found we should return status code 404 and message: User hobbies not found